### PR TITLE
UG: Add screenshots for Sessions

### PR DESCRIPTION
**Pull request by chwoozy**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -392,7 +392,15 @@ This is done as we do not except tutors to teach lessons beyond the day itself.
 </div>
 
 Example:
-* `add_session n/John Doe d/2021-01-01 t/18:00 k/120 s/Biology f/80`
+1. You would like to add a new session for **John Doe** on the date **2021-01-01** at **18:00** where you will teach **Biology** for **120** minutes and charge a fee of **$80**.
+2. Type in `add_session n/John Doe d/2021-01-01 t/18:00 k/120 s/Biology f/80` to add the session information into TutorBuddy and press <kbd>enter</kbd>.
```
Comment (enhao25) [inline docs/UserGuide.md]: Haha for <kbd>enter</kbd>, I think samuel previously mention that we should standardize, so I changed from "enter" over to "Enter" instead. I think you can do that as well ah.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -407,8 +415,15 @@ Arguments similar to `add_session` command except the following:
   e.g. `INTERVAL` = 7 for weekly sessions
 
 Example:
-* `add_rec_session n/John Doe d/2021-01-01 e/2021-01-29 b/7 t/18:00 k/120 s/Biology f/80` adds a weekly session that starts on
-2021-01-01 and ends on 2021-01-29
+1. You would like to add a new recurring session for **John Doe** on the date **2021-01-01** at **18:00** where you will teach
+   **Math** for **120** minutes and charge a fee of **$80** for each lesson. The session will occur every **7** days until **2021-01-29**.
+2. Type in `add_rec_session n/John Doe d/2021-01-02 e/2021-01-30 b/7 t/18:00 k/120 s/Math f/80` to add the session information into TutorBuddy and press <kbd>enter</kbd>.
```
Comment (enhao25) [inline docs/UserGuide.md]: Same comment as before.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -392,7 +392,15 @@ This is done as we do not except tutors to teach lessons beyond the day itself.
 </div>
 
 Example:
-* `add_session n/John Doe d/2021-01-01 t/18:00 k/120 s/Biology f/80`
```
Comment (JonahhGohh) [inline docs/UserGuide.md]: I think can add a <br> tag here?
image

File: docs/UserGuide.md (hunk 1 above)
Comment (JonahhGohh) [inline docs/UserGuide.md]: We decided to standardise to use upper case Enter instead of enter.

File: docs/UserGuide.md (hunk 2 above)
Comment (JonahhGohh) [inline docs/UserGuide.md]: Same here, Enter instead of enter

File: docs/UserGuide.md [hunk 4]
```diff
@@ -442,13 +465,16 @@ Arguments similar to `delete_session` command except the following:
...
-* This will split the recurring session into two recurring sessions that span the period exclusively
-before and after 2021-03-31 as shown below.
+1. You would like to delete an existing session from a recurring session for **John Doe** on the date **2021-01-16** at **18:00**.
+2. Locate the recurring session index to delete on the right of the session card.
+
+![delete_recurring_session_example](images/choonwei/delete_recurring_session.png)
+
+3. Using the index from step 2, type in `delete_rec_session n/John Doe i/1 d/2021-01-16` which deletes a single session from **John Doe**'s recurring session.
+4. You will now see that this splits the original recurring session into two recurring session from **2021-01-02** to **2021-01-09** and
+   another from **2021-01-23** to **2021-01-30**.
```
Comment (JonahhGohh) [inline docs/UserGuide.md]: should we follow the date format as displayed in the session card? i.e. 02 Jan 2021

**Review by JonahhGohh — COMMENTED** [review]
A few comments!

**Review by enhao25 — COMMENTED** [review]
LGTM. Only have a comment about changing enter to Enter for consistency with the rest. I think previously we were thinking that each example is done on a fresh new sample data (What I have done for students also), but I think yours is all done on a single instance with john doe already added inside also. Don't mind this ah, but I think u need to indicate this with a note after this part ah 
image

File: docs/UserGuide.md (hunk 2 above)
Comment (chwoozy) [reply docs/UserGuide.md]: Thanks for reminding! Made the changes.

File: docs/UserGuide.md (hunk 1 above)
Comment (chwoozy) [reply docs/UserGuide.md]: Thanks for reminding! Made the changes.

Comment (chwoozy) [reply docs/UserGuide.md]: Thanks for reminding! Made the changes.

File: docs/UserGuide.md (hunk 2 above)
Comment (chwoozy) [reply docs/UserGuide.md]: Thanks for reminding! Made the changes.

File: docs/UserGuide.md (hunk 4 above)
Comment (chwoozy) [reply docs/UserGuide.md]: Agreed! Made changes!

File: docs/UserGuide.md (hunk 3 above)
Comment (chwoozy) [reply docs/UserGuide.md]: What do you mean by this?

File: docs/team/yungweezy.md [hunk 5]
```diff
@@ -54,6 +53,6 @@ Given below is a summary of my contributions to the development of TutorBuddy.
         * Added sub-headers that groups commands into different categories to improve readability.
     * Developer Guide
         * Added implementation details for the `list` command along with the relevant sequence and activity diagram ([PR #100](https://github.com/AY2021S2-CS2103T-T11-1/tp/pull/100))
-        * Added implementation details for the `delete_rec_session` command along with the relevant sequence and activity diagram ([PR #303])(https://github.com/AY2021S2-CS2103T-T11-1/tp/pull/303)
+        * Added implementation details for the `delete_rec_session` command along with the relevant sequence and activity diagram ([PR #303])(https://github.com/AY2021S2-CS2103T-T11-1/tp/pull/303))
```
Comment (enhao25) [inline docs/team/yungweezy.md]: Haha I think for this part, the hyperlink is still not working tou, might need to change ([PR #303]) to [PR #303] instead.

File: docs/UserGuide.md [hunk 6]
```diff
@@ -369,6 +369,10 @@ Example:
 Please note that the following session commands can only be used after you have at least 1 student in TutorBuddy.
 Click [here](#423-managing-students) if you want to find out more on how to add your first student.
 
+<div markdown="block" class="alert alert-info">
+:information_source: Note that the examples in this section are done in a single instance with a student already existing in TutorBuddy.
```
Comment (enhao25) [inline docs/UserGuide.md]: Haha wait actually I think using single instance here might not be very understandable. Maybe using words like consecutive or something might be better? What do you think?

**Review by enhao25 — APPROVED** [review]
LGTM. Just a comment about the PPP and another one on probably making the explanation for the sessions to be of 1 instance better.

File: docs/UserGuide.md [hunk 7]
```diff
@@ -426,7 +445,15 @@ Format: `delete_session n/STUDENT_NAME i/SESSION_INDEX`
 </div>
 
 Example:
-* `delete_session n/John Doe i/1` deletes John Doe's **first** session based on the session list in the Tuition page
+1. You would like to delete an existing session for **John Doe** on the date **2021-01-01** at **18:00**.
```
Comment (enhao25) [inline docs/UserGuide.md]: Actually now that I tink about it. Maybe we want to have an example 2 where it deletes an entire recurring session. If not adding tip below that we can use this command to delete the entire session might be good also. What do you think.

**Review by enhao25 — COMMENTED** [review]
Sorry man, suddenly tot that maybe undering delete session, we should edit state or give example of delete_session effects on recurring session.

File: docs/UserGuide.md (hunk 3 above)
Comment (JonahhGohh) [reply docs/UserGuide.md]: Psps, i meant can add a br tag between the Tip and Example?
So:
teach lessons beyond...
"br tag here"
Example:

Comment (chwoozy) [reply docs/UserGuide.md]: Ah okay, have done the changes thank you!

File: docs/UserGuide.md (hunk 6 above)
Comment (chwoozy) [reply docs/UserGuide.md]: Agreed made some changes!

File: docs/UserGuide.md (hunk 7 above)
Comment (chwoozy) [reply docs/UserGuide.md]: Added a new example to `delete_session`

**Pull request merged**